aiida/cmdline/commands/cmd_profile.py

```diff
@@ -133,69 +135,49 @@ def profile_setdefault(profile):
 
 
 @verdi_profile.command('delete')
-@options.FORCE(help='to skip questions and warnings about loss of data')
-@click.option(
-    '--include-config/--skip-config',
-    default=True,
-    show_default=True,
-    help='Include deletion of entry in configuration file.'
-)
+@options.FORCE(help='Skip any prompts for confirmation.')
 @click.option(
-    '--include-db/--skip-db',
-    'include_database',
-    default=True,
-    show_default=True,
-    help='Include deletion of associated database.'
-)
-@click.option(
-    '--include-db-user/--skip-db-user',
```

Comment:
I suppose the tricky bit here is that this option would only make sense for the `psql_dos` storage. ^^ I'm fine removing it from the `verdi profile delete` command, although it might be useful in some cases (e.g. a user that relies on `verdi quicksetup` and keeps aggregating new Postgres users. 😅)

Reply:
Sure, but I think a better solution would be to then move all functionality of the quicksetup to the `core.psql_dos` storage plugin and then have the `delete` method check if the user of the storage that is being deleted matches that of one having been created by the quicksetup functionality, and then delete it if it no longer has any other databases. We shouldn't have to have a special option for this.

Reply:
Agree 👍 that would be a much sexier solution. ^^